app/assets/stylesheets/_bourbon-deprecate.scss

```diff
@@ -0,0 +1,19 @@
+@charset "UTF-8";
+
+/// Throws Sass warnings to announce library deprecations. People can disable
+/// these using the `$disable-bourbon-deprecation-warnings` variable.
```

Comment:
Yes, great point. I debated this because this current language aligns with Neat’s implementation (suite consistency), but the double negative is indeed a bummer. `output-bourbon-deprecation-warnings` is much more clear and straightforward.